app/bot/openai.go

```diff
@@ -52,11 +52,22 @@ func (o *OpenAI) OnMessage(msg Message) (response Response) {
 		return Response{}
 	}
 
-	if o.nowFn().Sub(o.lastDT) < 30*time.Minute && !o.superUser.IsSuper(msg.From.Username) {
-		log.Printf("[WARN] OpenAI bot is too busy, last request was %s ago, %s banned", time.Since(o.lastDT), msg.From.Username)
+	wtfContains := WTFSteroidChecker{
+		message: reqText}
+
+	if (o.nowFn().Sub(o.lastDT) < 30*time.Minute || wtfContains.ContainsWTF()) && !o.superUser.IsSuper(msg.From.Username) {
```

Comment:
nit: this if/else construction looks a little bit awkward. I would suggest an internal func, smth like "checkRequest" instead (with early return)